### DISCUSSION TITLE
Domain works but want to change to subdomein

### DISCUSSION THREAD

**Discussion by Dutchy-79** [homeassistant-apps/app-cloudflared#354] [Q&A] [answered]
Hi,

I registered a domain on cloudflare and use it to access HA. Working great.

Though instead of "myname.xx" I want to use ha.myname.xx instead.

I cannot find the right page that tells me how to use sub domain instead and how to setup the sub domain.

Please help me out,

Dutchy

**Comment by brenner-tobias** [comment, marked as answer]
This is quite easy. In the field "external_hostname" simply enter your subdomain and not your domain and restart the add-on.

Please let me know if this makes sense or if I am missing something.

**Comment by Dutchy-79** [comment]
Ty very much, that's it. Working great now.
